File: UI/desafio-dev-full/src/app/routes/home/components/Models/TituloModel.ts
import { ParcelaModel } from "./ParcelaModel";

export class TituloModel {
    Numero: number;
    NomeDevedor: string;
    CpfDevedor: string;
    PercentualJuros: number;
    PercentualMulta: number;
    Parcelas: ParcelaModel[] = [];


}

File: DesafioDevFull.Dominio/Models/TituloModel.cs
using System.Collections.Generic;

namespace DesafioDevFull.Dominio.Models
{
    public class TituloModel
    {
        public int Numero { get; set; }
        public string NomeDevedor { get; set; }
        public string CpfDevedor { get; set; }
        public decimal PercentualJuros { get; set; }
        public decimal PercentualMulta { get; set; }
        public IList<ParcelaModel> Parcelas { get; set; }
    }
}
